packages/SwingSet/src/vats/comms/remote.js

```diff
@@ -50,6 +50,8 @@ export function addRemote(state, name, transmitterID) {
     nextResolverIndex: state.identifierBase + 30,
     nextPromiseIndex: state.identifierBase + 40,
     transmitterID,
+    nextSendSeqNum: 1,
```

Comment:
Should we make these BigInts? These channels will be pretty long-lived, between chains. OTOH `2**53` and one message per second would get us a lifetime of half a billion years, so we're not in any immediate danger :). It'd be nice to use `Nat` for everything. Maybe we should schedule a swingset-wide conversion from Number to Nat for our counter values for some time in the future, and put it off until we have the bandwidth to change everything at once (and deal with the JSON/storage issues that will result).

Reply:
Nah.  The current intersection Nats & BigInts with serialization & deserialization is a world of pain and inconvenience, and in this case buys us nothing.